### DISCUSSION TITLE
How to change the Image of the Print Bed

### DISCUSSION THREAD

**Discussion by h2eder** [OrcaSlicer/OrcaSlicer#310] [Q&A]
Hi,

i have copied the Profile from the Artillery Printer to change it 
to a Tenlog TL-D3 Pro Profile. Works best so far, but it has the Artillery
Print Bed Image. Any Idea how i can change the Image to a given jpg or png?

Any Help would be appreciate.

**Comment by RaketenZonk** [comment]
Hi, you can add an .stl for the shape of the bed and an .svg for the texture to the folder where you put in all the other data about machines, filament and processes.
![grafik](https://user-images.githubusercontent.com/46133958/218722553-627dd07d-df81-4a1b-b268-c0b948fc946e.png)
You need to write the name of the files to your machine model file:
![grafik](https://user-images.githubusercontent.com/46133958/218723333-cffc2452-263b-4c95-938f-2bc373656757.png)

**Comment by h2eder** [comment]
Thanks for the fast Reply.

So, it's not so easy that i thought. 

I have only choose a Profile - my Artillery Sidewinder Profile - change some Values to the one for my Tenlog TL-D3 Pro and saved it as a new Printer...

Looks like thats not the correct Way to create a new Printer.

**Reply by RaketenZonk** [reply]
Ah ok, I see. Sorry, I thought you created a complete new profile.
In the printer settings there is also an option to import a texture and a model. For this you need to click on the "Set..." button next to "Printable area"
![grafik](https://user-images.githubusercontent.com/46133958/218758671-2b0b76f8-210c-4f14-ae4f-a7022b83e11c.png)

**Reply by h2eder** [reply]
Thanks, i tried that already but that has no Effect. The Image of the Bed don't change...

**Reply by Roysen66** [reply]
> Thanks, i tried that already but that has no Effect. The Image of the Bed don't change...

I have the same problem here. Any solution?

**Reply by TacticallyGrumpy** [reply]
I just went into the profile (BambuStudio-SoftFever>resouces>profiles>Prusa) I was using (mk3s) and changed the svg and stl to my SV06 stuff and just renamed it. I'm not going to get a Prusa so I'll only ever need it for my Sovol.

**Comment by jpcad64** [comment]
hello, I would like to know if someone has already configured a printer other than bambulab and changed the texture of the bed.
On the previous version, softfever 1.4.5. I managed to change it by modifying the file: Tronxy X5SA 400 Marlin Firmware.json in: C:\Users\toto\AppData\Roaming\OrcaSlicer\system\Tronxy\machine.
With the orcaslicer version it no longer works. If we modify the corresponding file, it resets it when launching the software... Does anyone have a tip?
![image](https://user-images.githubusercontent.com/47536014/227783855-b00e5560-60bb-4375-95e9-db8733757046.png)
